spec/fixtures/dcim/rack-role_1.json

```diff
@@ -0,0 +1,6 @@
+{
+    "id": 1,
```

Comment:
As I can see the indentation is a bit more than 2 spaces here.

Reply:
Yes, that's because the JSON was copied directly from the server's answer.